database/migrations/2022_08_19_183038_rename_gatwick_gnd_students_role.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class RenameGatwickGndStudentsRole extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::table('mship_role')->where(['name' => 'Gatwick Students'])->update(['name' => 'Gatwick GND Students']);
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        //
```

Comment:
I think it might be good to include the down migration just in case
```suggestion
        DB::table('mship_role')->where(['name' => 'Gatwick GND Students'])->update(['name' => 'Gatwick Students']);
```